kvbc/src/Replica.cpp

```diff
@@ -431,6 +432,10 @@ Replica::Replica(ICommunication *comm,
       secretsManager_{secretsManager},
       blocksIOWorkersPool_((replicaConfig.numWorkerThreadsForBlockIO > 0) ? replicaConfig.numWorkerThreadsForBlockIO
                                                                           : std::thread::hardware_concurrency()) {
+  bft::communication::CommStateControl::instance().setCommRestartCallBack([this]() {
+    this->m_ptrComm->Stop();
```

Comment:
style:
`m_ptrComm` -> comm_
`Start()` and `Stop()` -> lowercase

Reply:
fixed